core/src/web/toast.ts

```diff
@@ -18,6 +18,7 @@ export class ToastPluginWeb extends WebPlugin implements ToastPlugin {
     const toast = document.createElement('pwa-toast') as any;
     toast.duration = duration;
     toast.message = options.text;
+    toast.position = options.position;
```

Comment:
Since web toast doesn't really support the position, better not to add code that might look like it's implemented but broken
```suggestion
    
```